src/groth16/fr.rs

```diff
@@ -262,8 +262,7 @@ mod tests {
 
             let proof = prove(&qap, (&sigmag1, &sigmag2), &weights);
 
-            assert!(verify(
-                &qap,
+            assert!(verify::<CoefficientPoly<FrLocal>, _, _, _, _>(
```

Comment:
We should try to find a way to make this type inference work better, but we can leave all of these changes as they are for now.

Reply:
I agree, it is something we will handle when refactoring